### PR TITLE
Release 14.0.1

### PR DESCRIPTION
**Pull request by fheinecke**
* Fixed issue where Teleport Connect Kube terminal throws an internal server error [#32612](https://github.com/gravitational/teleport/pull/32612)
* Fixed `create_host_user_mode` issue with TeleportRole in the Teleport Operator CRDs [#32557](https://github.com/gravitational/teleport/pull/32557)
* Fixed issue that allowed for duplicate Access List owners [#32481](https://github.com/gravitational/teleport/pull/32481)
* Removed unnecessary permission requirement from PostgreSQL backend [#32474](https://github.com/gravitational/teleport/pull/32474)
* Added feature allowing for managing host sudoers without also creating users [#32400](https://github.com/gravitational/teleport/pull/32400)
* Fixed dynamic labels not being present on server access audit events [#32382](https://github.com/gravitational/teleport/pull/32382)
* Added PostHog events for discovered Kubernetes Apps [#32379](https://github.com/gravitational/teleport/pull/32379)
* Fixed issue where changing the cluster name leads to cluster being unaccessible [#32352](https://github.com/gravitational/teleport/pull/32352)
* Added additional logging for when the Teleport process file is not accessible due to a permission issue upon startup [#32348](https://github.com/gravitational/teleport/pull/32348)
* Fixed issue where the `teleport-kube-agent` Helm chart would created the same `ServiceAccount` multiple times [#32338](https://github.com/gravitational/teleport/pull/32338)
* Fixed GCP VM auto-discovery bugs [#32316](https://github.com/gravitational/teleport/pull/32316)
* Added Access List usage events [#32297](https://github.com/gravitational/teleport/pull/32297)
* Allowed for including only traits when doing a JWT rewrite for web application access [#32291](https://github.com/gravitational/teleport/pull/32291)
* Added `IneligibleStatus` fields for access list members and owners [#32278](https://github.com/gravitational/teleport/pull/32278)
* Fixed issue where the auth server was listed twice in the inventory of connected resources [#32270](https://github.com/gravitational/teleport/pull/32270)
* Added three second shutdown delay on on `SIGINT`/`SIGTERM` [#32189](https://github.com/gravitational/teleport/pull/32189)
* Add initial ServiceNow plugin [#32131](https://github.com/gravitational/teleport/pull/32131)

### CODE REVIEW

**Comment by fheinecke** [comment]
Labelling as "do not merge" pending merge of https://github.com/gravitational/teleport/pull/32595

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## 14.0.1 (09/26/23)
+
+* Fixed issue where Teleport Connect Kube terminal throws an internal server error [#32612](https://github.com/gravitational/teleport/pull/32612)
+* Fixed `create_host_user_mode` issue with TeleportRole in the Teleport Operator CRDs [#32557](https://github.com/gravitational/teleport/pull/32557)
+* Resources will now be refreshed after Connect My Computer setup [#32484](https://github.com/gravitational/teleport/pull/32484)
```
Comment (zmb3) [inline CHANGELOG.md:7]: You can remove this one. We haven't launched CMC yet.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+* Fixed issue that allowed for duplicate Access List owners [#32481](https://github.com/gravitational/teleport/pull/32481)
+* Added progress bar to the Connect My Computer setup screen [#32475](https://github.com/gravitational/teleport/pull/32475)
```
Comment (zmb3) [inline CHANGELOG.md:9]: Similarly, remove this one.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,31 @@
...
+* Removed unnecessary permission requirement from PostgreSQL backend [#32474](https://github.com/gravitational/teleport/pull/32474)
+* Added agent log storing to Connect My Computer [#32458](https://github.com/gravitational/teleport/pull/32458)
```
Comment (zmb3) [inline CHANGELOG.md:11]: Remove.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,31 @@
...
+* Added feature allowing for managing host sudoers without also creating users [#32400](https://github.com/gravitational/teleport/pull/32400)
+* Added compatibility information to Connect My Computer (#31951) [#32394](https://github.com/gravitational/teleport/pull/32394)
```
Comment (zmb3) [inline CHANGELOG.md:13]: Remove

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,31 @@
...
+* Fixed dynamic labels not being present on server access audit events [#32382](https://github.com/gravitational/teleport/pull/32382)
+* Added PostHog events for discovered Kubernetes Apps [#32379](https://github.com/gravitational/teleport/pull/32379)
+* Added agent removal support to Connect My Computer [#32369](https://github.com/gravitational/teleport/pull/32369)
```
Comment (zmb3) [inline CHANGELOG.md:16]: Remove

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,31 @@
...
+* Fixed issue where changing the cluster name leads to cluster being unaccessible [#32352](https://github.com/gravitational/teleport/pull/32352)
+* Added additional logging for when the Teleport process file is not accessible due to a permission issue upon startup [#32348](https://github.com/gravitational/teleport/pull/32348)
+* Fixed issue where the `teleport-kube-agent` Helm chart would created the same `ServiceAccount` multiple times [#32338](https://github.com/gravitational/teleport/pull/32338)
+* Fixed GCP VM auto-discovery bugs [#32316](https://github.com/gravitational/teleport/pull/32316)
+* Added Access List usage events [#32297](https://github.com/gravitational/teleport/pull/32297)
+* Implement waiting for Connect My Computer node to join cluster [#32295](https://github.com/gravitational/teleport/pull/32295)
```
Comment (zmb3) [inline CHANGELOG.md:22]: Remove

**Review by zmb3 — APPROVED** [review]
Looks good, I would just remove the connect my computer items.

**Review by camscale — APPROVED** [review]

**Pull request merged**